src/components/base/InternalButton/InternalButton.tsx

```diff
@@ -89,6 +90,7 @@ const UnstyledInternalButton = (props: InternalButtonProps) => {
       onMouseEnter={handleMouseEnter}
       onMouseLeave={handleMouseLeave}
       type={props.type ?? "button"}
+      form={props.form ? props.form : undefined}
```

Comment:
```suggestion
      form={props.form}
```

ternary seems redundant

Reply:
oh yeah